sdcm/stress_thread.py

```diff
@@ -128,11 +128,21 @@ def create_stress_cmd(self, node, loader_idx, keyspace_idx):
                           3]  # make sure each loader is targeting on datacenter/region
             first_node = first_node[0] if first_node else self.node_list[0]
             stress_cmd += " -node {}".format(first_node.ip_address)
-
-        stress_cmd += ' -errors skip-unsupported-columns'
-
+        stress_cmd = self._add_errors_option(stress_cmd, ['skip-unsupported-columns'])
```

Comment:
since when we are adding this unconditionally ?

Reply:
@fruch , since https://github.com/scylladb/scylla-cluster-tests/commit/b9043b4e998be1dbe3f7e411069c48ab5632ab07